docs/application/dotnet/guides/multimedia/audio.md

```diff
@@ -137,11 +137,51 @@ To query audio device information:
         ```
 
 
-        > **Note**   
-		> The initial running state of the connected device is `false`, which means the connected device is not running.
+        > **Note**
+        >
+        > The initial running state of the connected device is `false`, which means the connected device is not running.
+
+-   To utilize additional functionality for AudioDeviceType.UsbAudio and AudioDeviceIoDirection.Output device, use the [Tizen.Multimedia.AudioDevice](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.AudioDevice.html) class:
```

Comment:
@mijin85cho There are some functions only for device of 'UsbAudio and Output direction' in AudioDevice class. You can see remarks on that functions like 
        /// <remarks>
        /// This device should be <see cref="AudioDeviceType.UsbAudio"/> type and <see cref="AudioDeviceIoDirection.Output"/> direction."

I think that 'setting the "additional" information' is not proper, because it has the default variable for these setting -e.g. sample rate/format. Rather we can say it sets a particular value among values from the getter functions. More generally, so I wrote "to utilize additional functionality for the devices" and described in more detail below section. Please check this comment once again. Thanks.

Reply:
It seems fine to me, I'll revise it, thank you.